src/app/events/new/components/FormField.tsx

```diff
@@ -1,52 +1,53 @@
 import type { FieldError, UseFormRegister } from "react-hook-form";
 
 type ValidFieldNames =
-	| "eventName"
-	| "startsAt"
-	| "startsAtTime"
-	| "endsAt"
-	| "endsAtTime"
-	| "type"
-	| "imageUrl"
-	| "briefDescription"
-	| "fullDescription";
+  | "eventName"
```

Comment:
formatting fix?

Reply:
yeah, some biome / prettier configs are probably different between mine and luiza's environment, but we are trying to fix this